homeassistant/package_constraints.txt

```diff
@@ -1,6 +1,6 @@
 requests>=2,<3
 pyyaml>=3.11,<4
-pytz>=2017.02
+pytz>=2016.10
```

Comment:
Oh I guess not if pytz is already available in the container and it matches 2016.10